languages/csharp/exercises/concept/dictionaries/.docs/instructions.md

```diff
@@ -0,0 +1,109 @@
+In this exercise you'll be writing code to keep track of international dialling codes via an international dialing code dictionary.
+
+The dictionary uses an integer for its keys (the dialing code) and a string (country name) for its values.
+
+You have 11 tasks which involve dictionaries.
+
+### 1. Create a New Dictionary
+
+```csharp
+Dictionaries.GetEmptyDictionary();
+// empty dictionary
+```
+
+### 2. Create a Pre-populated Dictionary
+
+Create a dictionary which contains the following 3 dialing codes: "United States of America" which has a code of 1, "Brazil" which has a code of 55 and "India" which has a code of 91:
+
+```csharp
+Dictionaries.GetExistingDictionary();
+// 1 => "United States of America", 55 => "Brazil", 91 => "India"
+```
+
+### 3. Add a Country to an Empty Dictionary
+
+Add "United Kingdom" with a dialing code of 44:
```

Comment:
Could you phrase this in more general terms, as the function doesn't necessarily add the United Kingdom, that is just an example, right? So more like "Add a country code/country combination to the dictionary" or something similar. This also applies to the other task descriptions in this file.